docs/usage/getting-started.rst

```diff
@@ -0,0 +1,95 @@
+Getting Started
+===============
+
+Install Clowder
+---------------
+
+As covered in the main README section [1]_ , the installation process for Clowder is a breeze. 
+
+Create your first ClowdEnvironment
+----------------------------------
+
+Let's make a namespace to hold all the resources we'll be creating 
+
+``kubectl create ns jumpstart``
+
+``oc project jumpstart``
+
+That's the example namespace we'll be using for the rest of the guide. 
+
+Now we can drop in our first resource, a ClowdEnvironment. A ClowdEnvironment, or ClowdEnv, is a
+CustomResource that defines the environment our ClowdApp will utilize. The ClowdEnv defines what
+types of services our app may require and what source is providing those services. For our purposes,
+these services are set to 'local' mode and will spin up pods in the ``jumpstart`` namespace. 
+
+The API docs for ClowdEnvironments can be found on redhatinsights.github.io [2]_
+
+The full annotated ClowdEnv file can be found in the Clowder examples directory [3]_
+
+``oc apply -f https://raw.githubusercontent.com/RedHatInsights/clowder/master/docs/examples/clowdenv.yml``
+
+Clowder will pickup and apply that env resource. You may notice there are no pods running -- and
+that's correct. Let's see what the ClowdEnv does. 
+
+``oc get env env-jumpstart -o yaml``
+
+As you can see in the output, we have providers [4]_ for the different services, but they won't
+do anything until a ClowdApp asks for them specifically. 
+
+  Note: Clowder creates two new resources types that are easy to query. ``env`` for ClowdEnvs and
```

Comment:
This kind of implies the creation is happening due to the events on this page, perhaps it would be better to mention that "as part of the clowder installation....."

Reply:
Moved to the top install section